tests/nodes/primary_nodes/test_material.py

```diff
@@ -118,7 +118,7 @@ def test_deserialize_material_from_json() -> None:
     api_material = {
         "name": "my cool material",
         "component_count": 0,
-        "computational_forcefield_count": 0,
+        "computation_forcefield_count": 0,
```

Comment:
should be `computational_forcefield_count`

Reply:
we need to be careful with these, since we haven't really tried and they depend on how the back end responds.